contracts/SpokePool.sol

```diff
@@ -225,49 +222,130 @@ abstract contract SpokePool is Testable, Lockable, MultiCaller {
         });
         bytes32 relayHash = _getRelayHash(relayData);
 
+        _fillRelay(relayHash, relayData, relayerFeePct, maxTokensToSend, repaymentChain);
+    }
+
+    function _fillRelay(
```

Comment:
do you have a section for internal methods? should probs be located there.

Reply:
done